app/Jobs/PgBackrestRestoreJob.php

```
@@ -0,0 +1,439 @@
+    public function __construct(
+        public StandalonePostgresql $database,
+        public DatabaseRestore $restore,
+        public ?ScheduledDatabaseBackupExecution $execution = null,
+        public ?string $targetTime = null
+    ) {
+        $this->onQueue('high');
+    }
+
+    public function handle(): void
+    {
+        $server = $this->database->destination->server;
```

PHPMD (2.15.0)

Avoid unused local variables such as '$server'. (undefined)

(UnusedLocalVariable)